build.xml

```diff
@@ -27,6 +27,7 @@
 
   <!-- Classpath for building and running tests -->
   <path id="classpath-tests">
+    <pathelement location="src/test/resources"/>
```

Comment:
I am currently wondering why those changes are needed, as this PR does not contain any change in test code. Can you explain please?